### ISSUE TITLE
Python is an unnecessary dependency

### ISSUE THREAD

**Issue by kitten** [rogual/homebrew-neovim-dot-app#2]
I became aware of this dependency when I ran into issues, as homebrew installed another version of python on my Mac. Fortunately the python2 that is already installed on OS X is up to date (10.11) and sufficient. I ran into issues when starting neovim when homebrew installed the second python. I'm sure I could've fixed this, but this dependency is not necessary.

Scons installs and works just fine without the python dependency, as python is already present on OS X. If not then the scons brew formula would list it as a dependency surely.

All in all, the user should probably decide what version of python2 he wants to install.

**Comment by rogual** [comment]
@randy3k What do you think?

**Comment by randy3k** [comment]
In my use case, I installed a different version of python 3 along with the system python 2. And homebrew only picks up the python3 which appears first in the PATH variable. so requiring a homebrew python seems the easiest way to solve this issue.

There may be better ways to handle this. For the moment, if you don't want the homebrew python, you specify the option `--without-python` when you install.

**Comment by rogual** [comment]
I think @philplckthun has a valid concern as it does seem to be overkill to compile a whole new Python installation by default.

**Comment by randy3k** [comment]
I agree. The best strategy would possibly be doing something like version-checking - check if python 2 could be found on the system. However, given both `macvim` and `vim` @ homebrew require `python` by default, I believe it is not too harmful to leave it as is now.

**Comment by finn** [comment]
@philpl @rogual @randy3k I believe this change will fix this issue:

https://github.com/rogual/homebrew-neovim-dot-app/pull/10

More info in the PR.

**Comment by luqasz** [comment]
```==> Installing rogual/neovim-dot-app/neovim-dot-app
Warning: Calling 'depends_on :python' is deprecated!
Use 'depends_on "python"' instead.
/usr/local/Homebrew/Library/Taps/rogual/homebrew-neovim-dot-app/neovim-dot-app.rb:14:in `<class:NeovimDotApp>'
Please report this to the rogual/neovim-dot-app tap!```
